engine/graphics/src/opengl/graphics_opengl.cpp

```diff
@@ -410,7 +410,7 @@ static void LogFrameBufferError(GLenum status)
     static GraphicsAdapterFunctionTable OpenGLRegisterFunctionTable();
     static bool                         OpenGLIsSupported();
     static HContext                     OpenGLGetContext();
-    static int8_t          g_null_adapter_priority = 1;
+    static int8_t          g_null_adapter_priority = 2;
```

Comment:
Change name.
Also, perhaps use the ADAPTER_FAMILY_x setting to order the in a more robust manner.